ai-review/analytics/test-scorecard.sh

```
@@ -61,8 +86,14 @@ fi
 special_input="$special/"$'input\'\'&pipe|"\\\nline'
 mkdir "$special_input"
 cp -R "$here/fixtures/." "$special_input/"
-"$here/build-scorecard.sh" "$special_input" "$special/output" >/dev/null
-jq -e . "$special/output/v1/scorecard-manifest.json" >/dev/null
+if special_error=$("$here/build-scorecard.sh" "$special_input" "$special/output" 2>&1); then
+  echo "input path with a quote was not rejected" >&2
+  exit 1
+fi
+if [[ "$special_error" != *"cannot represent safely"* ]]; then
+  echo "special input path failed for the wrong reason: $special_input_error" >&2
```

Shellcheck (0.11.0)

[warning] special_input_error is referenced but not assigned.

(SC2154)